x-pack/plugins/maps/public/components/tooltip_selector/add_tooltip_field_popover.tsx

```diff
@@ -27,6 +27,8 @@ export type FieldProps = {
   name: string;
 };
 
+type FieldOptions = Array<EuiSelectableOption<{ value?: string }>>;
```

Comment:
value will always be provided and should not be optional.

Reply:
Good catch. Making `value` required actually highlights a bug in EUI that we'll want to fix.
For now I can do a safe cast and add a comment with the EUI issue link